scripts/check_compliance.py

```diff
@@ -1085,6 +1087,12 @@ def init_github(args):
     else:
         github_pr = None
 
+    # Get the shippable build number, useful to find logs
+    build_number = os.environ.get('BUILD_NUMBER')
```

Comment:
Maybe double quotes just to be consistent with next lines (though this script is a bit inconsistent in general).

Reply:
done